autogluon/contrib/enas/enas.py

```diff
@@ -303,6 +303,44 @@ def __repr__(self):
         reprstr += ')\n'
         return reprstr
 
+    def export(self, path, epoch=0, remove_amp_cast=True):
+        """Export HybridBlock to json format that can be loaded by
+        `gluon.SymbolBlock.imports`, `mxnet.mod.Module` or the C++ interface.
+
+        .. note:: When there are only one input, it will have name `data`. When there
+                  Are more than one inputs, they will be named as `data0`, `data1`, etc.
+
+        Parameters
+        ----------
+        path : str
+            Path to save model. Two files `path-symbol.json` and `path-xxxx.params`
+            will be created, where xxxx is the 4 digits epoch number.
+        epoch : int
+            Epoch number of saved model.
```

Comment:
Could we add mini code example here showing how to export and reload exported model?